utils/https.c

```diff
@@ -63,7 +63,9 @@ status_t send_https_msg(char const *host, char const *port, char const *api, con
     ta_log_error("http(s) read response error, return code %d\n", ret);
     goto exit;
   }
-
+#ifdef ENABLE_ENDPOINT_TEST
```

Comment:
Does it make sense to dump results here?

Reply:
I think it is better to remove the dump results here, we will create the new issue to get HTTP result from callback.